### DISCUSSION TITLE
How to hash password?

### DISCUSSION THREAD

**Discussion by TheresaRoss** [prisma/prisma#16614] [Q&A] [answered]
I don't want to store passwords directly in db, does Prisma offer anything to help hash passwords? and how to Authenticate?

**Comment by technotip** [comment, marked as answer]
You can use bcrypt for hashing the password. https://www.npmjs.com/package/bcrypt

**Reply by vladi-stevanovic** [reply]
👋 Hi @TheresaRoss, did you perhaps have a chance to check Technotip's recommendation? Let us know if you still have any questions! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
